woff1/tools/validator/validator.py

```diff
@@ -1326,9 +1323,8 @@ def _testMetadataEncoding(data, reporter):
     """
     if _shouldSkipMetadataTest(data, reporter):
         return False, False
-    metadata = unpackMetadata(data, parse=False)
+    metadata = unpackMetadata(data, parse=False).decode()
```

Comment:
Binary data needs to be decoded in order to be able to perform string comparison in Python 3.